tests/CppUTestExt/MockSupportTest.cpp

```diff
@@ -909,6 +939,388 @@ TEST(MockSupportTest, customObjectWithFunctionComparatorThatFailsCoversValueToSt
     CHECK_EXPECTED_MOCK_FAILURE_LOCATION(failure, __FILE__, __LINE__);
 }
 
+TEST(MockSupportTest, customTypeOutputParameterSucceeds)
+{
```

Comment:
Please remove this kind of comments from the tests. (all of them).

If you want to seperate the AAA sections, just use an empty line. No need to name them.